pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -308,7 +313,101 @@ func (r *ReconcileAccountClaim) Reconcile(request reconcile.Request) (reconcile.
 		if err != nil {
 			return reconcile.Result{}, err
 		}
+	}
+
+	// Set awsAccountClaim.Spec.AwsAccountOU
+	if unclaimedAccount.Spec.AwsAccountOU == "" {
```

Comment:
Can we wrap all the logic under this into a function?

`if unclaimedAccount.Spec.AwsAccountOU == "" {
    placeAccountInOU(...)
}`

Reply:
I split the logic into better components. I personally prefer the way it is now over dumping it all inside a function but let me know if you disagree.

Reply:
It clutters up the accountclaim reconcile function. We recently cleaned up the account reconciler to better organize the steps. Opinions from others?

Reply:
Done